### PR TITLE
s5wesley Dockerfile

### PR DESCRIPTION
**Pull request by s5wesley**

### CODE REVIEW

**Review by s6dolapo — APPROVED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -1 +1,88 @@
+# Set up ARG variable to allow the user to specify the tag version at build time
+ARG UBUNTU_VERSION=22.04
```
Comment (ekenmoe) [inline Dockerfile]: On this ARG UBUNTU_VERSION, we are suppose to call this argument at run time, providing the OS version, therefore I believe we shouldn't specified the OS version on the Dockerfile.

**Review by ekenmoe — COMMENTED** [review]
Done messing up with your work. Lol

**Review by s5wesley — COMMENTED** [review]
great catch

**Comment by ekenmoe** [comment]
Looks good, please merge.

**Pull request closed without merging**